helm/e2esetup-certs-chart/templates/calico-cert.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: "core.giantswarm.io/v1alpha1"
```

Comment:
This one can be removed. It's no longer used since we switched to latest certs. Both v18 and v19 use them now.

Reply:
What about `calico-etcd-client-cert.yaml`?

Reply:
This one we use to allow calico access etcd.